flixel/FlxCamera.hx

```diff
@@ -1525,6 +1525,27 @@ class FlxCamera extends FlxBasic
 		setScale(scaleX, scaleY);
 	}
 	
+	/**
+	 * Helper function to make sure a point will use integer coordinates after
+	 * applying scale.
+	 */
+	public inline function floorPoint(point:FlxPoint):FlxPoint
```

Comment:
I do think they might be useful elsewhere. Seems like in a lot of places we're flooring before camera scale is applied, resulting in final positions that aren't actually integers. Not sure how many places this might be needed, but I'm guessing more than just FlxTilemap. I originally used these in FlxObject and FlxSprite too, but removed them until I can figure out the jittering issue.